sql/table.go

```diff
@@ -30,7 +31,7 @@ import (
 
 func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 	desc := structured.TableDescriptor{}
-	desc.Name = p.Table.String()
+	desc.Name = strings.ToLower(p.Table.String())
```

Comment:
If we're going to be case-insensitive we should at least be case-preserving. We can lowercase the name when constructing the metadata key but we should preserve the original capitalization in the descriptor (and use this capitalization in all output)